restart-local.sh

```
@@ -0,0 +1 @@
+git pull && docker compose down -f compose.local.yml && docker compose up -d --build --remove-orphans -f compose.local.yml
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)